Composer/packages/adaptive-flow/src/adaptive-flow-renderer/widgets/ActionCard/CardComment.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import React from 'react';
+import { TruncatedText } from '@bfc/ui-shared';
+
+type CardCommentProps = {
+  comment?: string;
+};
+
+const styles = {
+  container: css`
+    background-color: #fff4ce;
+    padding: 8px;
+    padding-bottom: 10px;
+    border-radius: 2px;
+    position: relative;
+  `,
+  fold: css`
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    height: 0;
+    width: 0;
+    border-top: 4px solid #ded2a7;
```

Comment:
Replace hex colors in this file

Reply:
These yellow colors don't exist in the fabirc npm packages yet.